File: Back/src/ProEventos.Infra/Entities/Palestrante.cs
using System.Collections.Generic;

namespace ProEventos.Infra.Entities
{
    public class Palestrante
    {
       public int Id { get; set; } 
       public string Nome { get; set; }
       public string Descricao { get; set; }
       public string ImagemUrl { get; set; }
       public string Telefone { get; set; }
       public string Email { get; set; }
       public IList<RedeSocial> RedesSociais { get; set; }
       public IList<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}

File: Front/ProEventos-App/src/app/models/Palestrante.ts
import { RedeSocial } from "./RedeSocial";
import { Evento } from "./Evento";

export interface Palestrante {
  id: number;
  nome: string;
  descricao: string;
  imagemUrl: string;
  telefone: string;
  email: string;
  redesSociais: RedeSocial[];
  palestrantesEventos: Evento[];
}
